DAS/tests/mqtt_test.py

```diff
@@ -5,24 +5,34 @@
 from das_data_generator import send_csv_data, send_fake_data
 
 # Arguments
-parser = argparse.ArgumentParser(description='DAS MQTT python script', add_help=True)
-parser.add_argument('-t', '--time', action='store', type=int, default=1, help="length of time to send data")
-parser.add_argument('-r', '--rate', action='store', type=float, default=0.5, help="rate of data in seconds")
-parser.add_argument('--host', action='store', type=str, default="localhost", help="address of the MQTT broker")
-parser.add_argument('--port', action='store', type=int, default=1883, help="address of the MQTT broker")
-parser.add_argument('--username', action='store', type=str, help="username for the MQTT broker (optional)")
-parser.add_argument('--password', action='store', type=str, help="password for the MQTT broker (optional)")
-parser.add_argument('-f', '--file', action='store', type=str, help="The csv file to replay. If not specified, makes up data.")
-parser.add_argument('-s', '--speed', action='store', type=float, default=1, help="Replay speed multiplier")
-parser.add_argument('-j', '--jump', action='store', type=int, default=0, help="Starts replaying from a specified time (in seconds)")
+parser = argparse.ArgumentParser(
+    description='DAS MQTT python script', add_help=True)
+parser.add_argument('-t', '--time', action='store', type=int,
+                    default=1, help="length of time to send data")
+parser.add_argument('-r', '--rate', action='store', type=float,
+                    default=0.5, help="rate of data in seconds")
+parser.add_argument('--host', action='store', type=str,
+                    default="localhost", help="address of the MQTT broker")
+parser.add_argument('--port', action='store', type=int,
+                    default=1883, help="address of the MQTT broker")
+parser.add_argument('--username', action='store', type=str,
+                    help="username for the MQTT broker (optional)")
+parser.add_argument('--password', action='store', type=str,
+                    help="password for the MQTT broker (optional)")
+parser.add_argument('-f', '--file', action='store', type=str,
+                    help="The csv file to replay. If not specified, makes up data.")
+parser.add_argument('-s', '--speed', action='store',
+                    type=float, default=1, help="Replay speed multiplier")
+parser.add_argument('-j', '--jump', action='store', type=int, default=0,
+                    help="Starts replaying from a specified time (in seconds)")
 
 
 def start_publishing(client, args):
     print("start")
     client.publish("power_model/start")
     client.publish("start")
 
-    send_data_func = lambda data: client.publish("data", data)
+    def send_data_func(data): return client.publish("data", data)
```

Comment:
If we're switching from a lambda to a function (any particular reason?) probably put the return statement on a new line.

Reply:
I did not do that, it appears as if the atom linter did it automatically. I'll change it back to a lambda as its much nicer